azkaban-common/src/main/java/azkaban/jobExecutor/utils/process/AzkabanProcess.java

```diff
@@ -122,12 +122,7 @@ public void run() throws IOException {
       errorGobbler.awaitCompletion(5000);
 
       if (exitCode != 0) {
-        final String output =
```

Comment:
This info should be in the logs already. Also, it wasn't used before because toString/getMessage methods of ProcessFailureException were not overwritten.